Frameworks/CoreText/CTRun.mm

```diff
@@ -117,7 +116,13 @@ void CTRunGetPositions(CTRunRef run, CFRange runRange, CGPoint* outPositions) {
         runRange.length = curRun->_range.length;
     }
 
-    memcpy(outPositions, &curRun->_glyphOrigins[runRange.location], sizeof(CGPoint) * runRange.length);
+    if (runRange.location != 0) {
+        UNIMPLEMENTED();
+        return;
+    }
+
+    outPositions->x = curRun->_xPos;
```

Comment:
>outPositions [](start = 4, length = 12)

I am not even sure if this is useful information.  We are essentially returning one glyph position.  Need a TODO to implement the real version.  What didn't work here with DWrite?